src/main/java/nikki/NikkiException.java

```diff
@@ -0,0 +1,7 @@
+package nikki;
+
+public class NikkiException extends Exception {
+    public NikkiException(String errorMessage) {
```

Comment:
Could consider adding a header comment to describe what kind of exceptions are being handled by this class/method